scripts/build/custom_platforms.js

```diff
@@ -604,7 +604,11 @@ module.exports = {
         'path': 'extension/chromium-mv3',
         'expires': '10 days',
         'configuration': {
-            'removeRulePatterns': CHROMIUM_BASED_EXTENSION_PATTERNS,
+            'removeRulePatterns': [
+                ...CHROMIUM_BASED_EXTENSION_PATTERNS,
+                // https://github.com/AdguardTeam/FiltersCompiler/issues/231
```

Comment:
Maybe comments are redundant? It is (will be) described in the syntax docs.

Reply:
the main reason why this file was made `.js` (instead of `.json`) is to have an ability to add comments. so I believe such comments are fine

Reply:
Every line must be commented even if all info exists in docs?)